sw/airborne/firmwares/rotorcraft/navigation.c

```diff
@@ -75,7 +75,7 @@ PRINT_CONFIG_VAR(NAVIGATION_FREQUENCY)
 #endif
 
 #define CLOSE_TO_WAYPOINT (15 << INT32_POS_FRAC)
-#define CARROT_DIST (12 << INT32_POS_FRAC)
+#define CARROT_DIST (200 << INT32_POS_FRAC)
```

Comment:
Maybe not, but it is an issue that this distance is quite small (and not configurable). If the reference model is not very aggressive, it will seriously limit the speed and acceleration towards waypoints that are far away. It is also not configurable...

Reply:
I agree that it should be configurable, but not for changing the previous behavior with such a large value. It will make the carrot jump on waypoints in a range of 200 m if I'm not wrong (and if ref traj is not used, which is not usually the case I agree).

Reply:
Ok I agree, I'll make it configurable